releasenotes/notes/0.10.2/bugfix-vf2-parallel-edges-f8be125e255753f7.yaml

```diff
@@ -3,12 +3,15 @@ fixes:
   - |
     Fixes the output of :func:`~retworkx.is_subgraph_isomorphic`
     if the input graphs have parallel edges.
+    Fixed `#429 <https://github.com/Qiskit/retworkx/issues/429`__
```

Comment:
This line is making the docs build fail

Reply:
Oops my bad, fixed the typo in: https://github.com/Qiskit/retworkx/pull/434/commits/b7127af68b36dfcbcaa4d9396b4fb1b84742b083